src/DebuggerModel/DebugSession.class.st

```diff
@@ -211,9 +211,7 @@ DebugSession >> isContextPostMortem: selectedContext [
 	| suspendedContext |
 	suspendedContext := interruptedProcess suspendedContext.
 	suspendedContext ifNil: [ ^ false ].
-	(suspendedContext == selectedContext)
-		ifTrue: [ ^ false ].
-	^ (suspendedContext findContextSuchThat: [:c | c sender == selectedContext]) isNil
+	^ (suspendedContext hasContext: selectedContext) not.
```

Comment:
It looks good, but is there a test for this functionality?

Reply:
Yes, in IsContextPostMortemTests>>#testIsContextPostMortem

Reply:
ok!